src/main.rs

```diff
@@ -66,16 +75,24 @@ enum Cli {
     /// Pay a number of bones to a given address. Note that 1 HNT is
     /// 100_000_000 bones
     Pay {
+        /// Whether to use Ledger
+        #[structopt(short = "l", long = "ledger")]
+        ledger: bool,
+
+        /// Whether to parse amount as bones
+        #[structopt(short = "b", long = "bones")]
+        bones: bool,
+
         /// Wallet to use as the payer
         #[structopt(short = "f", long = "file", default_value = "wallet.key")]
         files: Vec<PathBuf>,
 
         /// Address of the payee
         address: String,
 
-        /// Number of bones to send
-        #[structopt(name = "bones")]
-        amount: u64,
+        /// Amount to send (assumed as HNT)
+        #[structopt(name = "amount")]
+        amount: String,
```

Comment:
This feels out of scope of this PR. But if it's really necessary to change the meaning of `amount` from bones to HNT, how about `amount: Decimal,` from the crate [`rust_decimal`](https://crates.io/crates/rust_decimal)? `Decimal` should (needs fact-checking) be able to represent HNT without any floating-point rounding errors when scaled to bones. And since `Decimal` implements `from_str()`, it can be used here without any manual call to parse it.

Reply:
This was a UX request made by Cokes. It seemed reasonable to me as I think most people think in HNT. I've taken your advice though and spun out a Hnt struct with recent commits